packages/common/base/src/Crossmint/Crossmint.ts

```diff
@@ -0,0 +1,13 @@
+export type CrossmintConfig = {
+    apiKey: string;
+    jwt?: string;
+    overrideBaseUrl?: string;
+};
+
+export class Crossmint {
+    constructor(public config: CrossmintConfig) {}
+
+    setJwt(jwt: string | undefined) {
```

Comment:
Can you add a typedoc? I think we're going to start requiring these for SDK reference autogeneration. PRobably cursor can write it for you

Reply:
the docs arent generated from the types?